templates/home/index.html.twig

```diff
@@ -1,7 +1,52 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Studentai{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
-{% endblock %}
+<ul class="list-group m-4">
+    <li class="list-group-item list-group-item-info">Studentai</li>
+    {% for student in students %}
+        <li class="list-group-item">
+            <a href="{{ path('student', {name: student['student'] , project: student['project'] }) }}">{{ student['student'] | escape}}</a>
+            (
+            <span class="badge">Mentorius</span> {{ student['mentors'] | join(', ') | escape }}
```

Comment:
:+1: Malonu, kad atsižvelgi į kelkis mentorius.

```suggestion
            <span class="badge">Mentorius</span> {{ student.mentors | join(', ') }}
```
Bet Symfony turi įjungtą `autoescape`, tai papildomai rašyti `| escape` nebūtina
Be to Twig palaiko ir alternatyvią (su tašku) sintaksę (kuri prie sudėtingesnių projektų gali palengvinti kodo skaitomumą)